__tests__/deinit.test.js

```diff
@@ -68,6 +68,9 @@ describe('Keybase bot deinitialization', () => {
     await alice.deinit()
     await bob.deinit()
 
+    // Also give a couple seconds for the processes to fully deinit
+    await timeout(3000)
```

Comment:
I was thinking about this with one of my other tests, and I think the problem with picking a specific timeout amount is that it risks being too low (and you get occasional failures/flakes) or too high (and you're waiting an extra few seconds on every test.  Given that there is a global timeout, it seems like you want the success case to exit as fast as possible, in which case I think I prefer this kind of thing:

```
while (await countProcessesMentioning(aliceHomeDir) > 0) {
  await timeout(100)
}
```

either way though this is fine with me.

Reply:
That's a good point – I'm gonna think about this more, I think we could maybe extract the about logic out into a function, e.g., `waitUntilThisConditionIsSatisfied(...)`.